terraform/modules/lambda_s3/variables.tf

```
@@ -64,7 +64,7 @@ variable "runtime" {
 
 variable "lambda_source_path" {
   type    = string
-  default = "./lambda/"
+  default = "../lambda/"
```

Does this need to now be 
```suggestion
  default = "../lambdas/"
```

---

needs to be `../src/lambdas/`